### PR TITLE
Add sample crops ml code

### PR DESCRIPTION
**Pull request by Subash-Lamichhane**

### CODE REVIEW

**Comment by Tpuljak** [comment]
@Subash-Lamichhane please sign your commit.

You can do that with:
```
git rebase HEAD~1 --signoff
git push -f
```

**Comment by Subash-Lamichhane** [comment]
@Tpuljak Thanks for help,
I have signed the commit.
Could you check this PR once again.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,37 @@
-This repository contains a README file sample for Daytona Samples and the MIT License.
+<!-- This repository contains a README file sample for Daytona Samples and the MIT License.
 
 It can be used as a template to create sample repositories that can be added into [Daytona](https://github.com/daytonaio/daytona).
 
-Once you finish your sample and it gets merged, you can open a PR in the Daytona repo and submit the sample into the [index file](https://github.com/daytonaio/daytona/blob/main/hack/samples/index.json).
+Once you finish your sample and it gets merged, you can open a PR in the Daytona repo and submit the sample into the [index file](https://github.com/daytonaio/daytona/blob/main/hack/samples/index.json). -->
```
Comment (Tpuljak) [inline README.md]: Remove all comments from the README

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Subash-Lamichhane) [reply README.md]: I have made the changes.

**Pull request merged**